pkg/epforwarder/epforwarder.go

```diff
@@ -40,6 +43,14 @@ var passthroughPipelineDescs = []passthroughPipelineDesc{
 		defaultBatchMaxContentSize:    20e6,
 		defaultBatchMaxSize:           pkgconfig.DefaultBatchMaxSize,
 	},
+	{
+		eventType:                     EventTypeNetworkDevicesMetadata,
+		endpointsConfigPrefix:         "network_devices.metadata",
```

Comment:
should `network_devices.metadata` be `network_devices.metadata.` (note the `.`)? based on the other usage it seems like this is necessary.

Reply:
Oh good catch, yes it is necessary

Reply:
Good catch. Thanks, it's fixed here https://github.com/DataDog/datadog-agent/pull/8140/commits/d640bca3ef0a6b00622dbbd072daa5be816de3eb

I previously only used `DD_SITE=datad0g.com` to be able to send data to staging.

@gh123man @djova Could you have another look ?

Reply:
It's working with:
```
network_devices:
  metadata:
    dd_url: network-devices.datad0g.com
```

Reply:
Yes, `DD_SITE=datad0g.com` then it will default to `hostnameEndpointPrefix` + `DD_SITE`, which in that case would be `network-devices.datad0g.com`. 

Looks good now!